pkg/ingress_flags/flags.go

```diff
@@ -37,7 +37,11 @@ import (
 	klog "k8s.io/klog/v2"
 )
 
-func parseFlags() (bool, *controller.Configuration, error) {
+// TODO: We should split the flags functions between common for all programs
+// and specific for each component (like webhook, controller and configurer)
```

Comment:
```suggestion
// and specific for each component (like webhook, controller and configure)
```

Reply:
nah, configurer is the name of the component per architectural doc :) I think we can leave as is right now (configurer is the component that configures nginx)

Reply:
got it